rust-toolchain.toml

```diff
@@ -0,0 +1 @@
+toolchain.channel = "1.78"
```

Comment:
what is the difference between .toml and previous rust-toolchain

Reply:
That format is obsolete it seems, as it didn't work out of the box. I didn't bother to look into it and started using the newer one (it's a few years old now)